src/screens/PinScreen.js

```diff
@@ -284,35 +299,43 @@ class PinScreen extends React.Component {
       );
     };
 
-    const renderPinDigits = () => (
+    const safeBiometryMessage = () => (
       <View
         style={{
           flex: 1,
           alignItems: 'center',
+          width: '100%',
+          justifyContent: 'space-between',
+          marginTop: 32,
+          marginBottom: 16,
```

Comment:
![image](https://github.com/user-attachments/assets/68111c3f-4a71-4dd6-b0d2-ddd3484ad4a5)

This is the screen that gets rendered, right?

Can you center the Spinner and the text?

Reply:
Done